code/game/objects/items/robot/robot_items.dm

```diff
@@ -36,119 +36,49 @@
 	name = "hugging module"
 	icon_state = "hugmodule"
 	desc = "For when a someone really needs a hug."
-	var/mode = 0 //0 = Hugs 1 = "Hug" 2 = Shock 3 = CRUSH
-	var/ccooldown = 0
+	var/mode = 0 //0 = Hug, 1 = Shock
```

Comment:
Make these defines--`MODE_HUG` and `MODE_SHOCK`.

Reply:
done